recipe/recipe.yaml

```diff
@@ -34,61 +38,110 @@ requirements:
+        - if exist %PREFIX%\Library\include\siscone\defines.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\circulator.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\area.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\ranlux.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\quadtree.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\protocones.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\momentum.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\hash.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\vicinity.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\split_merge.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\siscone.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\siscone_error.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\reference.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\spherical\geom_2d.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\spherical\vicinity.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\spherical\split_merge.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\spherical\siscone.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\spherical\protocones.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\spherical\momentum.h (echo exists) else (exit /B)
+        - if exist %PREFIX%\Library\include\siscone\spherical\hash.h (echo exists) else (exit /B)
+        - cd examples
+        - ..\build\examples\Release\siscone_main.exe
+        - ..\build\examples\Release\siscone_area.exe
+        - ..\build\examples\Release\sample.exe
+        - ..\build\examples\Release\siscone_test.exe
+        - if: build_platform == target_platform
+          then: ..\build\examples\Release\times.exe
+        - ..\build\examples\Release\spherical.exe
     files:
       source:
-        - examples/main.cpp
-        - examples/options.cpp
-        - examples/options.h
-        - examples/area.cpp
-        - examples/sample.cpp
-        - examples/test.cpp
-        - examples/times.cpp
-        - examples/spherical.cpp
+        - if: win
+          then:
+            - build/examples/Release/siscone_main.exe
+            - build/examples/Release/siscone_area.exe
+            - build/examples/Release/sample.exe
+            - build/examples/Release/siscone_test.exe
+            - build/examples/Release/times.exe
+            - build/examples/Release/spherical.exe
```

Comment:
By copying over the prebuilt binaries that both increases the metadata size of the package and also then doesn't test the user experience of being able to build using the installed versions of the library, which is that these example tests are meant to do. Is there any issue with using the installed version of the library?

Reply:
I don't yet know of a way to invoke MSVC's compiler like gcc on the commandline. It otherwise requires a CMakelists.txt or a `<name>.vcxproj` file so that msbuild knows what to do. So, comparatively, for these tests it's much more cumbersome in windows to achieve what you say but is not technically impossible.

This also means, in general, the user experience on windows is vastly altered compared to unix-likes. 

Thus, I copied the built executables and at least showed they run and are portable for now.

Reply:
I'd also say that the binaries are much smaller than the test data we ship around with the package, which dominates the size of the package anyway.

Reply:
Indeed, hilariously, the linux-64 package is 1.0 MB and the win-64 package is 940 KB. So from the size perspective this is a non-issue.

Demonstration-of-usability-wise it's sorta handled by looking at the linux and mac tests, and I imagine the majority of the C++ developer user base will be in unix-likes. I expect windows users to mostly be folks that are eventually using fastjet-cxx-python and fastjet (once those exist on windows).